fe/fe-core/src/main/java/com/starrocks/journal/JournalWriter.java

```diff
@@ -177,11 +177,11 @@ protected void abortJournalTask(JournalTask task, String msg) {
 
     private boolean shouldCommitNow() {
         // 1. check if is an emergency journal
-        if (currentJournal.getBetterCommitBeforeTime() > 0) {
-            long delayMillis = (System.nanoTime() - currentJournal.getBetterCommitBeforeTime()) / 1000000;
-            if (delayMillis >= 0) {
-                LOG.warn("journal expect commit before {} is delayed {} mills, will commit now",
-                        currentJournal.getBetterCommitBeforeTime(), delayMillis);
+        if (currentJournal.getBetterCommitBeforeTimeInNano() > 0) {
+            long delayNanos = System.nanoTime() - currentJournal.getBetterCommitBeforeTimeInNano();
+            if (delayNanos >= 0) {
```

Comment:
feels like 0 is too strict, since it's nano time, it can easily cause a warning log, how about  > 500ms, we print a log?

Reply:
Fixed